packages/redshift/sodasql/dialects/redshift_dialect.py

```diff
@@ -117,6 +118,33 @@ def __get_cluster_credentials(self):
 
         return cluster_creds['DbUser'], cluster_creds['DbPassword']
 
+    def __query_table(self, table_name):
+        query = f"""
+        SELECT *
+        FROM {table_name}
+        LIMIT 1
+        """
+        return query
+
+    def sql_test_connection(self) -> Union[Exception, bool]:
```

Comment:
Do we need these functions here? RedshiftDialect extends PostgresDialect anyway 🤔

Reply:
good catch!